.github/workflows/ci.yml

```diff
@@ -10,6 +10,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install packages
-      run: npm install
+      run: yarn install
     - name: Run tests
-      run: npm run test:unit
+      run: yarn test:unit
+    - name: build
```

Comment:
I think we didn't have a build step in before. It only adds a relatively small amount of time so I'm not opposed to keeping it. In fact, this step is called `build` so I can see why this may have been added lol

Reply:
Yeah, I thought it might be worthwhile to run a build in this context (only partially because of the name - but also if we wanted more data in case a deploy fails). If the builds start to take too long, we can discuss removing it / renaming the workflow...